dsl_parser/rel_graph.py

```diff
@@ -495,7 +495,7 @@ def _build_and_update_node_instances(ctx,
         new_instances_num = current_instances_num
 
     new_containers = []
-    for _ in range(new_instances_num):
+    for _ in range(int(new_instances_num)):
```

Comment:
Is this for when providing the number of instances via a secret?

Reply:
I think this is not via secret. (sorry, I couldn't catch what 'via a secret' mean)

This is for when inputting value(1, 2, ..., etc) in CLI（"cfy deplyments create -b blueprint_id --input value"） or WebUI.  Error Messege is "Internal server error".
The value is defined by a blueprint (inputs: retry_after: default: 30, etc) . Even if I defined value's type (inputs: retry_after: type: integer, etc), inputting value raise the same error.

[FYI : Enviroment]
Cloudify 4.2 , CentOS 7.x

Reply:
[CFY-7653] Cloudify manager hits an internal error when providing an integer as a deployment input - GigaSpaceshttps://cloudifysource.atlassian.net/browse/CFY-7653